tests/integ/modin/series/test_getitem.py

```diff
@@ -201,3 +201,15 @@ def test_df_getitem_with_multiindex(
             lambda ser: ser[key],
             check_index_type=False,
         )
+
+
+@sql_count_checker(query_count=1)
+def test_series_getitem_lambda_series():
```

Comment:
good to add but I try to understand this case won't go with your code change right?

Reply:
Oh this is code I am moving - this was in the dataframe getitem test file instead of series, so I moved it